test/json_unittest.cc

```diff
@@ -143,6 +163,8 @@ TEST(TrivialToStringTest, ArrayOneElement) {
 TEST(TrivialParseTest, ObjectEmpty) {
   GuardJsonException([](){
     json::value v = json::Parser::FromString("{}");
+    EXPECT_TRUE(v->Is<json::Object>());
+    EXPECT_EQ(json::ObjectType, v->type());
     EXPECT_TRUE(v->As<json::Object>()->empty());
```

Comment:
For the nonempty case, can you also check that you can extract a `json::value` using `at` (i.e., `obj->at("f")`) and that `obj->at("g")` throws `std::out_of_bounds`?

Reply:
Done.